lapack/lapack64/lapack64.go

```diff
@@ -725,7 +748,7 @@ func Ormlq(side blas.Side, trans blas.Transpose, a blas64.General, tau []float64
 // If lwork is -1, instead of performing Ormqr, the optimal workspace size will
 // be stored into work[0].
 func Ormqr(side blas.Side, trans blas.Transpose, a blas64.General, tau []float64, c blas64.General, work []float64, lwork int) {
-	lapack64.Dormqr(side, trans, c.Rows, c.Cols, a.Cols, a.Data, max(1, a.Stride), tau, c.Data, max(1, c.Stride), work, lwork)
+	lapack64.Dormqr(side, trans, c.Rows, c.Cols, len(tau), a.Data, max(1, a.Stride), tau, c.Data, max(1, c.Stride), work, lwork)
```

Comment:
This is actually a bug fix. Since the signature of Ormqr does not specify k, it must be determined from somewhere and the length of tau is the natural (and only) option. That's also why the documentation change.

I thought about including a change in this PR to require `len(tau) == k` in the Lapack call graph of all these QR-related functions (we do it in some functions already) but the change was quite pervasive, so I'll leave it for later.